pyproject.toml

```diff
@@ -72,7 +72,7 @@ extend-exclude = [
 
 [tool.ruff.lint]
 # See https://docs.astral.sh/ruff/rules/
-select = ["B", "C4", "DTZ", "E", "F", "G", "I", "PERF", "PGH", "PT", "PYI", "RUF", "S", "T20", "W"]
+select = ["B", "C4", "DTZ", "E", "F", "G", "I", "PERF", "PGH", "PT", "PYI", "RUF", "S", "T20", "UP", "W"]
```

Comment:
Maybe we should add this to the template. What do you think?

Reply:
I agree. I think it'll be much easier in the future if we turn this one earlier.